### PR TITLE
Adding expressive SSML support

### PR DESCRIPTION
**Pull request by leonrch**
## THIS IS A WORK IN PROGRESS PLEASE DON'T MERGE 

Adding expressive SSML support

### CODE REVIEW

File: manifest.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ declared-services:
 applications:
 - services:
   - text-to-speech-service
-  name: text-to-speech-demo
+  name: text-to-speech-demo-lr
```
Comment (germanattanasio) [inline manifest.yml]: make sure you revert this change

File: public/js/demo.js [hunk 2]
```diff
@@ -152,9 +153,22 @@ $(document).ready(function() {
...
+			else {
+				$('#textArea').val(englishText);
+				var en_accent = voice.substring(0, 5);
+				if(en_accent === 'en-US') {
+					$('#ssmlArea').val(usEnglishSSML);
+				}
+				else if(en_accent === 'en-GB') {
+					$('#ssmlArea').val(ukEnglishSSML);
+				}
+			}
```
Comment (germanattanasio) [inline public/js/demo.js]: check the indentation

**Comment by germanattanasio** [comment]
The Pull request looks good. Let me know when you want me to merge it

File: public/js/constants.js [hunk 3]
```diff
@@ -12,20 +13,22 @@ var spanishText = "Consciente de su patrimonio espiritual y moral, la Unión est
...
-
+	
 window.SPEECH_SYNTHESIS_VOICES = {
     voices: [
     {
         "name": "en-US_AllisonVoice", 
         "language": "en-US", 
         "customizable": true, 
-        "gender": "female", 
+        "gender": "female, expressive",
```
Comment (nfriedly) [inline public/js/constants.js]: This looks wrong... is that the API that the server will be using? Is it too late to change it?

Comment (germanattanasio) [inline public/js/constants.js]: I just saw this. I'm wondering what @jsstylos thinks about this :bowtie:

Comment (nfriedly) [inline public/js/constants.js]: @daniel-bolanos sent out an email to Juraj Figura & Arthur Kantor,asking for confirmation on what the REST API will look like. We'll go from there.

Comment (leonrch) [inline public/js/constants.js]: Unfortunately, there is no **expressive** field is exposed yet by the service. The easiest way to verify it is to issue the following CURL command:
 curl -u "{username}":"{password}" "https://stream.watsonplatform.net/text-to-speech/api/v1/voices".
- There is no difference between Allison (expressive voice) and other (non-expressive) voices in   the returned JSON. 
- Yes, I think it is too late to change it now. We can change it as soon as the TTS service will provide with the **expressive** field.
